src/storekit.cs

```diff
@@ -525,34 +546,59 @@ interface SKStoreProductViewController {
 		void LoadProduct (StoreProductParameters parameters, [NullAllowed] Action<bool,NSError> callback);
 	}
 
+	[Mac (11,0), NoTV, NoWatch]
+	[BaseType (typeof (NSObject))]
+	[Model (AutoGeneratedName = true)]
```

Comment:
did you add `AutoGeneratedName = true`
iirc that's a potential breaking change in existing models
c.c. @rolfbjarne

Reply:
@spouliot added as per @rolfbjarne request

Reply:
hmm... is this a new type ? (then it needs it)
or does it look new because the diff is hard to read (and the code just moved) ? (then I don't think it should be there)

Reply:
It was moved. So no need.
```suggestion
	[Model]
```

Reply:
Please don't resolve a conversation you have not started. Makes things harder to review again.